### PR TITLE
README: micro-edits [ci skip]

### PR DESCRIPTION
**Pull request by olleolleolle**
Thank you for offering this work as Open Source.

This PR makes copy edits to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ $ ab -n 10000 -c 100 http://localhost:4423/api/hello.json
 Command `k8rb`
 --------------
 
-Keight.rb provices `k8rb`.
+Keight.rb provides `k8rb`.
 
 * `k8rb project myapp1` creates new project.  
   (Note: This is equvarent to `boilerpl8 github:kwatch/keight-ruby myapp1`.)
```
Comment (marcinruszkiewicz) [inline README.md]: This should be `equivalent`, right? (also 4 lines below)

Comment (olleolleolle) [inline README.md]: Thanks, edited.

Comment (marcinruszkiewicz) [inline README.md]: 👍

**Comment by kwatch** [comment]
Thank you very much! And sorry for late reply.

**Pull request merged**